src/main/java/spoon/reflect/meta/impl/ModelRoleHandlers.java

```diff
@@ -2350,7 +2350,7 @@ public <T, U> void setValue(T element, U value) {
 		}
 	}
 
-	static class CtSynchronized_BODY_RoleHandler extends SingleHandler<CtSynchronized, CtBlock<? extends Object>> {
```

Comment:
This change also removes redundant upper bounds in many places. Is this a shortcoming of the sniper printer or intentional?

Reply:
None of them. This class is generated from CtGenerationTest which does not use sniper.

Reply:
Kind of scary because I cannot think of any reason why this happens, but I'm fine with it.

Reply:
Git bisect found the culprit:
```
0fcb28878671d90daede513326bff96274068b32 is the first bad commit
commit 0fcb28878671d90daede513326bff96274068b32
Author: I-Al-Istannen <I-Al-Istannen@users.noreply.github.com>
Date:   Tue Apr 19 22:46:53 2022 +0200

    refactor: Rewrite type adaption (#4215)

binary search found first bad commit
```

I plead the fifth and refuse any comment. I don't event want to think about why this changed whether wildcard bounds are printed or not, that sounds like a quite unimportant hole for more masochistic moods.

Reply:
:rofl: